examples/halo_test/halo_test.f90

```diff
@@ -334,6 +340,9 @@ subroutine test_div_haloZ()
 #else
       logical, parameter :: global = .false.
 #endif
+      integer :: i1, in ! I loop start/end
```

Comment:
Note to self - check if `in` causes issues as a reserved keyword

Reply:
I also don’t like in. This was there from before and so far as not created any issue. I would be happy to change it, but we might have to change also also the rest accordingly for consistency i.e. i1, in j1, jn k1 kn. Shall we use double letter like ii1 iin?

Reply:
`ifirst`/`ilast`? Pretty sure this is my fault so will make the change.